examples/chart/teleport-kube-agent/values.schema.json

```diff
@@ -253,6 +264,63 @@
             "type": "object",
             "default": {}
         },
+        "extraLabels": {
+            "$id": "#/properties/extraLabels",
+            "type": "object",
+            "required": [
+                "config",
+                "deployment",
+                "pod",
+                "serviceAccount"
```

Comment:
I don't think these are actually required? All `extraLabels` are inherently optional, right, so there shouldn't be any required parameters.